### ISSUE TITLE
Youtube Unblocker Links Not Working

### ISSUE THREAD

**Issue by Browncha023** [Browncha023/EndInternetCensorship#13]
Fixing Soon...

**Comment by PitifulAttempt** [comment]
the no cookie thing works when out of the proxy but not when in

**Comment by Browncha023** [comment]
Yeah, that is the problem. They used to work but they haven't for like a month now.

**Comment by Browncha023** [comment]
I'm closing all issues as I am completely ending support for EIC, I've already phased out of supporting it for a while now. It has not received any updates in months and the github port will be left in an unfinished state. Thank you for using EIC!
